test/test_google_cloud_manage.py

```diff
@@ -1197,8 +1156,13 @@ def test_get_project_members(test_cloud_manager):
 
     test_cloud_manager._authed_session.post.return_value = _fake_response(
         200, faked_reponse_body)
-    members = test_cloud_manager.get_project_members()
-    assert members == ['test@gmail.com', 'test@example.net']
+    members = test_cloud_manager.get_project_membership()
+    for mem in members:
+        assert mem in [
+        GooglePolicyMember("user", "test@gmail.com"),
```

Comment:
style nitpick: fix indentation (GooglePolicyMemb... and following lines should be bumped in one level)